server/sonar-server/src/main/java/org/sonar/server/computation/ws/TaskFormatter.java

```diff
@@ -175,4 +200,32 @@ Long computeExecutionTimeMs(CeQueueDto dto) {
     }
     return system2.now() - startedAt;
   }
+
+  private final class CeActivityDtoToTask implements Function<CeActivityDto, WsCe.Task> {
+    private final ComponentDtoCache cache;
+
+    public CeActivityDtoToTask(ComponentDtoCache cache) {
+      this.cache = cache;
+    }
+
+    @Override
+    @Nonnull
+    public WsCe.Task apply(@Nonnull CeActivityDto input) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) The "input" parameter nullability is different in the superclass method, and that should not be changed. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/coding_rules#rule_key=squid%3AS2638)
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) The return value nullability of this method is different in the superclass, and that should not be changed. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/coding_rules#rule_key=squid%3AS2638)